docs/how_to/hypervisors/vagrant_libvirt.md

```diff
@@ -0,0 +1,54 @@
+# Vagrant Libvirt #
+
+This driver enables to trigger tests using libvirtd daemon. It is
+based
+on [libvirt](https://github.com/vagrant-libvirt/vagrant-libvirt)'s
+plugin for vagrant.
+
+## Basic Options ##
+
+With the libvirt daemon on the same host than the running test command
```

Comment:
the host running the test is referred to in our docs as 'the beaker coordinator', so this might be clarified by saying:

    Once you've setup the libvirt daemon on your beaker coordinator, you can use the vagrant_libvirt hypervisor
    by providing beaker with a configuration similar to this: